src/main/java/nextstep/subway/domain/section/Sections.java

```diff
@@ -14,36 +13,33 @@
 @Embeddable
 public class Sections {
     @OneToMany(mappedBy = "line", cascade = {CascadeType.PERSIST, CascadeType.MERGE}, orphanRemoval = true)
-    private List<Section> sectionList = new ArrayList<>();
+    private List<Section> sectionList;
+
+    public Sections() {
+        this.sectionList = new ArrayList<>();
+    }
 
     public List<Station> getAllStations() {
         List<Station> stations = new ArrayList<>();
-        sectionList.stream().forEach(section -> section.push(stations));
+        sectionList.forEach(section -> section.push(stations));
 
         return stations;
     }
 
     public boolean hasAnyMatchedDownStation(Station upStation) {
-        return sectionList.stream()
-                .filter(section -> section.hasDownStation(upStation))
-                .findAny()
-                .isPresent();
+        return sectionList.stream().anyMatch(section -> section.hasDownStation(upStation));
```

Comment:
팀 컨벤션마다 다를 수는 있겠지만
stream 이후로 개행을 해주는 것이 가독성이 좋다고 생각합니다 😄 
```suggestion
        return sectionList.stream()
                .anyMatch(section -> section.hasDownStation(upStation));
```